tests/integration/ha/test_storage.py

```diff
@@ -61,33 +64,29 @@ async def test_storage_reuse_after_scale_down(
             "reuse of storage can only be used on deployments with persistent storage not on rootfs deployments"
         )
 
-    # scale-down to 1 if multiple units
-    unit_ids = get_application_unit_ids(ops_test, app)
-    if len(unit_ids) > 1:
-        for unit_id in unit_ids[1:]:
-            await ops_test.model.applications[app].destroy_unit(f"{app}/{unit_id}")
-
-        await ops_test.model.wait_for_idle(
-            apps=[app],
-            status="active",
-            timeout=1000,
-            wait_for_exact_units=1,
-            idle_period=IDLE_PERIOD,
-        )
-    else:
-        # wait for enough data to be written
-        time.sleep(60)
-
     writes_result = await c_writes.stop()
 
+    # scale up to 2 units
+    await ops_test.model.applications[app].add_unit(count=1)
+    await ops_test.model.wait_for_idle(
+        apps=[app],
+        status="active",
+        timeout=1000,
+        wait_for_exact_units=2,
+    )
+
```

Comment:
I believe we could move the line `writes_result = await c_writes.stop()` to here, so that enough writes get written by then (the scale up substitutes the need for time.sleep, this will be useful in the future when we'd like to ensure that the data itself has been reused)

Reply:
done